src/tools/rustfmt/src/config/options.rs

```diff
@@ -18,7 +18,7 @@ pub enum NewlineStyle {
     Auto,
     /// Force CRLF (`\r\n`).
     Windows,
-    /// Force CR (`\n).
+    /// Force CR (`\n`).
```

Comment:
Please don't make this change here, make it upstream in rustfmt instead. The lint isn't warn-by-default so it shouldn't break CI.

Reply:
Sure, I've dropped this change.